component_text/absrel.md

```diff
@@ -0,0 +1,97 @@
+# aBSREL
```

Comment:
I see. I had originally added it to @stephenshank's fork so I wouldn't have to email him all the time..